reading_json.py

```diff
@@ -0,0 +1,53 @@
+import os
+import shutil
+import json
+import sys
+
+
+# FINISH TESTING & DEBUGGING
+#
+#
+
+def main():
+
+    # getting cmd_line_args in format:
+    # "python3 reading_json.py file.json img_dir output_dir"
+    cmd_line_args = sys.argv
+    print(cmd_line_args)
```

Comment:
Lets try not to leave debug code in 😄

Reply:
Oh that's supposed to be just sample usage for anyone reading the code. If that's unnecessary, I can remove it.